src/components/appmodalcartselect.main.jsx

```diff
@@ -48,15 +54,38 @@ class AppModalCartSelectMain extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      orgEamData: undefined,
       selectedCart: '0',
       selectedCartName: optionCarts[0].optionName,
     };
     this.continueCart = this.continueCart.bind(this);
+    this.fetchOrganizationData = this.fetchOrganizationData.bind(this);
     this.handleCartChange = this.handleCartChange.bind(this);
   }
 
   componentDidMount() {
-    this.continueCart();
+    // this.continueCart();
+    this.fetchOrganizationData();
+  }
+
+  fetchOrganizationData() {
+    cortexFetch(`/eam?zoom=${zoomArray.join()}`, {
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: localStorage.getItem(`${Config.cortexApi.scope}_oAuthTokenEam`),
+      },
+    })
+      .then(res => res.json())
+      .then((res) => {
+        const orgEamData = res._authorizationcontexts[0]._element.find(element => element.name === Config.cortexApi.scope.toUpperCase());
```

Comment:
If I logged in I still have empty option cart:
<img width="572" alt="screen shot 2019-01-22 at 11 13 49 am" src="https://user-images.githubusercontent.com/34708483/51524527-f50f5200-1e36-11e9-8f61-84a8019e73db.png">

I noticed all will works correctly if use 'MOBEE' instead 'VESTRI'   from our config: res._authorizationcontexts[0]._element.find(element => element.name === 'MOBEE');

I think backend settings should be updated to 'VESTRI'?

Reply:
@petro97 The API should be returning an organization of the same scope that we have in the storefront. It's similar to how we select the "scope" in the API when doing a storefront shopping call to the backend. In a real b2b flow for the APIs, there would be multiple organizations returned. The storefront would then filter through the list and get the data for the scope which the store is configured for (Our store uses Vestri, but the current data on the backend has Mobee). Let me know if you have more questions on this and we can chat over slack 😃